pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[tool.ruff]
+lint.select = ["PL", "YTT", "BLE", "I", "UP", "A", "COM", "T10", "EXE", "TID", "F401", "F402", "F821"]
+
+lint.ignore = ["E501", "PLR2004", "COM812", ]
+
+target-version = "py310"
+
+extend-exclude = [
+    "docker/alternator-dns",
+]
+
+force-exclude = true
+line-length = 160
```

Comment:
Here it is.
The default is 88 according to the official docs [1].
160 is too big for opening 2 vertical windows on a screen with 1920 width.
My eyes can afford me +-120 symbols in this case.
Another bigger font size is 112.
So, if we could have here at max 112 or 110 symbols. It would be great.

[1] https://docs.astral.sh/ruff/settings/#line-length

Reply:
I was playing around with this figure for the exact same reason.

it was 120 with autopep8, but it wasn't that strict, like for comment

Reply:
it's now 112